typings/index.d.ts

```diff
@@ -3083,6 +3085,20 @@ declare module 'discord.js' {
     cache?: boolean;
   }
 
+  interface HTTPAttachmentData {
+    name: string;
```

Comment:
This is missing a property:
```suggestion
    attachment: string;
    name: string;
```
See: https://github.com/discordjs/discord.js/blob/99e2ca408b0ead894c2d91bc2902c392a3635fb0/src/structures/APIMessage.js#L303